graylog2-web-interface/src/views/logic/aggregationbuilder/AggregationWidget.ts

```diff
@@ -19,49 +19,51 @@ import { Map } from 'immutable';
 import isDeepEqual from 'stores/isDeepEqual';
 import isEqualForSearch from 'views/stores/isEqualForSearch';
 import type { QueryString, TimeRange } from 'views/logic/queries/Query';
+import type { FiltersType } from 'views/types';
 
 import AggregationWidgetConfig from './AggregationWidgetConfig';
 
 import Widget from '../widgets/Widget';
 
 export default class AggregationWidget extends Widget {
-  constructor(id: string, config: AggregationWidgetConfig, filter?: string, timerange?: TimeRange, query?: QueryString, streams: Array<string> = []) {
-    super(id, AggregationWidget.type, config, filter, timerange, query, streams);
+  // eslint-disable-next-line default-param-last
+  constructor(id: string, config: AggregationWidgetConfig, filter?: string, timerange?: TimeRange, query?: QueryString, streams: Array<string> = [], filters?: FiltersType) {
+    super(id, AggregationWidget.type, config, filter, timerange, query, streams, filters);
   }
 
   static type = 'AGGREGATION';
 
   static defaultTitle = 'Untitled Aggregation';
 
   static fromJSON(value) {
-    const { id, config, filter, timerange, query, streams } = value;
+    const { id, config, filter, timerange, query, streams, filters } = value;
 
-    return new AggregationWidget(id, AggregationWidgetConfig.fromJSON(config), filter, timerange, query, streams);
+    return new AggregationWidget(id, AggregationWidgetConfig.fromJSON(config), filter, timerange, query, streams, filters);
   }
 
   toBuilder() {
-    const { id, config, filter, timerange, query, streams } = this._value;
+    const { id, config, filter, timerange, query, streams, filters } = this._value;
 
-    // eslint-disable-next-line no-use-before-define
-    return new Builder(Map({ id, config, filter, timerange, query, streams }));
+    // eslint-disable-next-line no-use-before-define,@typescript-eslint/no-use-before-define
```

Comment:
The `typescript-eslint` hint should replace the previous one:

```suggestion
    // eslint-disable-next-line @typescript-eslint/no-use-before-define
```